src/components/charts/LineChart.tsx

```diff
@@ -0,0 +1,185 @@
+import React from "react"
+import merge from "deepmerge"
+
+import {
+  ChartWrapper,
+  RecordSet,
+  RecordEntry,
+  ChartProps,
+} from "./ChartWrapper"
+
+export interface LineChartProps extends ChartProps {
```

Comment:
Thinking about this a bit more, since these are actually just self-contained types that define arguments, perhaps they should all be local and not extend the base `ChartProps` ? Just thinking that tracking this over time will get very gnarly and maybe best to keep things separate?

Reply:
I think they do share a few common keys like `title` and `subtitle` so extending a common interface encourages code reuse and makes documentation a bit easier